### PR TITLE
short description of what/why to learn coffeescript

### PR DESCRIPTION
**Pull request by dsudia**
Fulfills need for ABOUT doc for CoffeeScript. Happy to get feedback on it!

### CODE REVIEW

File: docs/ABOUT.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+JavaScript is a great language, but not everyone likes writing in it. Thus, people and organizations have started to write languages that transpile(get changed from one syntax to another) into JavaScript, that they feel are nicer to write in than JavaScript. CoffeeScript is one example, another is Microsoft's TypeScript.
```
Comment (kytrinyx) [inline docs/ABOUT.md]: `s/transpile(get/transpile (get/`

File: docs/ABOUT.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+> CoffeeScript is a little language that compiles into JavaScript. Underneath that awkward Java-esque patina,
+> JavaScript has always had a gorgeous heart. CoffeeScript is an attempt to expose the good parts of JavaScript
+> in a simple way.
+>
+> The golden rule of CoffeeScript is: "It's just JavaScript". The code compiles one-to-one into the equivalent JS,
+> and there is no interpretation at runtime. You can use any existing JavaScript library seamlessly from CoffeeScript
+> (and vice-versa). The compiled output is readable and pretty-printed, will work in every JavaScript runtime, and tends
+> to run as fast or faster than the equivalent handwritten JavaScript.
+
+You should learn CoffeeScript if you like what you get out of JavaScript, but don't like what you have to put in.
```
Comment (kytrinyx) [inline docs/ABOUT.md]: The second half of this sentence suggests that if you _do_ like JavaScript, you shouldn't learn CoffeeScript. I think that people could get a lot out of learning things that transpile to JavaScript, even if they like JavaScript.

**Comment by kytrinyx** [comment]
Thanks! @exercism/javascript Do you have any thoughts about this?

**Comment by dsudia** [comment]
Great notes, thanks! Maybe instead something like, "You should learn CoffeeScript if you like programming in JavaScript, and want to experiment with a different way to write it. With transpiled version of JavaScript (ECMA 2015, TypeScript, etc) becoming more popular, this is a valuable skill for a coder." What do you think?

**Comment by matthewmorgan** [comment]
Looks good, and +1 for playing up the value of learning to transpile JS.  One comment I would make is that @dsudia your proposed addition contains a typo: 'version' should be 'versions'.

**Comment by rchavarria** [comment]
Yep, looks good to me too.

I've also heard that some ECMAScript 2015 features were inspired by CoffeeScript (classes, arrow functions, destructuring,... not sure which of them exactly). So one could see CoffeeScript as a predecessor of ECMAScript 2015, and that's nice!

**Comment by dsudia** [comment]
Ok, made the changes! I also made the description of transpiling more specific and accurate.

File: docs/ABOUT.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+JavaScript is a great language, but not everyone likes writing in it. Thus, people and organizations have started to write languages that transpile(transpiling keeps the same level of abstraction, compiling decreases the level of abstraction) into JavaScript, that they feel are nicer to write in than JavaScript. CoffeeScript is one example, another is Microsoft's TypeScript.
```
Comment (kytrinyx) [inline docs/ABOUT.md]: This needs a space before the opening parenthesis.

**Comment by kytrinyx** [comment]
I like the text now. Just one typo that I missed the first time around.

**Comment by dsudia** [comment]
fixed the typo!

**Comment by kytrinyx** [comment]
Thanks!

**Pull request merged**